### PR TITLE
Use json5 for parsing 

### PR DESCRIPTION
**Pull request by gkiely**
Allows using things like single quotes, unquoted properties. More here: https://json5.org/

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "istanbul": "^0.4.5",
     "markdown": "^0.5.0",
     "mocha": "^6.2.2",
-    "should": "^13.2.3"
+    "should": "^13.2.3",
+    "json5": "^2.1.3"
```
Comment (haoxins) [inline package.json:37]: this should be in deps, not devDeps

Comment (gkiely) [reply package.json]: Fixed.

**Comment by haoxins** [comment]
`+ gulp-file-include@2.2.0`

**Comment by gkiely** [comment]
@haoxins thanks for approving so fast!

**Pull request merged**